lexers/testdata/gitignore.expected

```diff
@@ -0,0 +1,5 @@
+[
+  {"type":"Error","value":"test\n\n"},
```

Comment:
You will need to add a rule to match everything that is not a comment, otherwise it will output error tokens like these.

Reply:
Whats the best way to write such a rule? Can I say, everything that don't match is text?

Reply:
Yep

Reply:
I tried it for some time to create a RegEx that match all line that not start with #, includes empty lines and works with Go, but I failed.

Reply:
Rules are tried in order, so you should just need something like `[^#]+`